config/settings.yml

```diff
@@ -24,6 +24,9 @@ defaults: &defaults
   interactions:
     must_be_follower: false
     must_be_following: false
+  reserved_usernames:
+    - 'admin'
+    - 'support'
```

Comment:
I added "admin" and "support" as default entries because those are two account names that I wouldn't want people to steal.